.genx/eslint.config.mjs

```diff
@@ -0,0 +1,20 @@
+import globals from 'globals';
+import pluginJs from '@eslint/js';
```

Comment:
We publish shared eslint/prettier settings so we don't have to redefine them. Using them here would be a bit fiddly, because we'd want to automatically upgrade their versions in our upgrade CI job, but that wouldn't work as things stand. Ok to have it like that for now I'd say.

https://github.com/genesislcap/foundation-ui/tree/master/packages/tooling/build